netctl/netctl.go

```diff
@@ -401,3 +402,14 @@ func dumpList(ctx *cli.Context, list []map[string]interface{}) {
 	os.Stdout.Write(content)
 	os.Stdout.WriteString("\n")
 }
+
+func showVersion(ctx *cli.Context) {
```

Comment:
btw please don't take it personally if any of the above came out too strong or aggressive. I just wish that we consider all the sides and make sure that we introduce new features in a maintainable fashion.